js/game.js

```diff
@@ -93,7 +95,13 @@ function fadeTables(tables) {
  */
 function showFinalScore() {
     gameOver.classList.remove('hidden')
-    gameOver.innerHTML = '<p>Game Over!</p><p class="final-score">You whacked ' + score + ' rats</p>'
+    if (score === 1) {
+        gameOver.innerHTML = '<p>Game Over!</p><p class="final-score">You whacked ' + score + ' rat</p>'
```

Comment:
inside your if have a variable called `msg` which changes in each statement, then set the html after the ifs using:
```
gameOver.innerHTML = '<p>Game Over!</p><p class="final-score">' + msg + '</p>'
```

Reply:
fixed, code dry